controllers/dislikes-controller.js

```diff
@@ -0,0 +1,92 @@
+import dislikesDao from "../database/dislikes/dislikes-dao.js";
+import booksDao from "../database/books/books-dao.js";
+
+const userDislikesBook = async (req, res) => {
```

Comment:
should the following async functions have awaits?

Reply:
Yes they should! I will change them! (These are for postman test, not used on react, but yeah they need)